### DISCUSSION TITLE
Storage size definition helm installation

### DISCUSSION THREAD

**Discussion by whebz** [datalust/seq-tickets#1497] [Help] [answered]
Hello,

I am trying seq for the first time and it's running in our k8s cluster (AKS), and I was wondering how can i define storage through config.yaml?

thanks

**Comment by whebz** [comment, marked as answer]
I found the answer.
Please ignore it

**Comment by KodrAus** [comment]
Hi @whebz :wave: Glad you found the answer. For anybody else coming here you can set storage through [the `persistence` property](https://github.com/datalust/helm.datalust.co/blob/ee04fa1a48a900080c4841ccffa925cf3f133b15/charts/seq/values.yaml#L105) in your Helm configuration.
